cmp/chart/ChartModel.js

```diff
@@ -38,14 +38,31 @@ export class ChartModel extends HoistModel {
     }
 
     /**
-     * @param {Object} config - Highcharts configuration object for the managed chart. May include any
-     *      Highcharts opts other than `series`, which should be set via `setSeries()`.
+     * Set the Highcharts configuration.
+     *
+     * @param {Object} config - Highcharts configuration object. May include any
+     *      Highcharts options other than `series`, which should be set via `setSeries()`.
+     *
+     *      See also {@see applyHighChartsConfig} for a method that will allow updating individual
```

Comment:
@lbwexler is applyHighChartsConfig supposed to be updateHighChartsConfig in this comment?

Reply:
fixed